src/Class Specs/Class_sport.jsx

```diff
@@ -496,38 +467,6 @@ class Sport extends React.Component {
         <Doughnut data={datas} options={options} />
       </div>
     );
-
-    return <></>;
-
-    //return createElement(Chart, { donutCenterRender: names }, "a");
-    /*
-    let data = createElement(
-      "div",
-      { className: "InnerData" },
-      <>
-        <Chart donutCenterRender={names}>
-          <ChartSeries>
-            <ChartSeriesItem
-              type="donut"
-              data={vals}
-              categoryField="kind"
-              field="share"
-            />
-            <ChartSeriesLabels
-              color="#acb"
-              background="none"
-              content={labels}
-            />
-          </ChartSeries>
-          <ChartLegend visible={true} />
-        </Chart>
-      </>
-    );
-    */
-    /*
-    return data;
-    return <></>;
```

Comment:
What was the main issue with this comment out code.

Reply:
The graphs were build twice, one using a somewhat nicer donuts graphs, while the other uses the basic, the commented out code is for donuts that enables words to be creating inside of them, with that it then required a key or some type of agreement to use. It was left.